pkg/daemon/ceph/osd/volume.go

```diff
@@ -858,16 +858,12 @@ func GetCephVolumeLVMOSDs(context *clusterd.Context, clusterInfo *client.Cluster
 			continue
 		}
 		var osdFSID string
-		store := "bluestore"
 		for _, osd := range osdInfo {
 			if osd.Tags.ClusterFSID != cephfsid {
 				logger.Infof("skipping osd%d: %q running on a different ceph cluster %q", id, osd.Tags.OSDFSID, osd.Tags.ClusterFSID)
 				continue
 			}
 			osdFSID = osd.Tags.OSDFSID
-			if osd.Type == "journal" {
```

Comment:
This could potentially break old clusters running with Filestore. It's such a small amount of code that I wouldn't mind keeping it up. What do you think?

Reply:
I thought since v1.3 support for filestore was removed and that all of them were required to convert to bluestore OSDs. How would filestore still be in use?

Reply:
We removed the support to deploy new Filestore OSDs but we "can" (not 100%) still activate them and that's what that code helps with.

Reply:
The [1.3 upgrade guide](https://rook.io/docs/rook/v1.3/ceph-upgrade.html#converting-legacy-osds) indicated that filestore osds would no longer be managed by rook. So I would think this is dead code. Sure, it doesn't hurt to keep if it actually is doing something, but doesn't appear to be.

Reply:
Ok!

Reply:
btw I left some of the placeholder code for the storeType since `cstore` is coming...

Reply:
Yes, `seastore` is coming :)

Reply:
I was afraid that was a typo :)